src/shared/components/QueryBuilder/domain/types.ts

```diff
@@ -48,6 +48,7 @@ export enum FilterPartKind {
 export type Suggestion = {
   value: string;
   label: string;
+  description?: string;
```

Comment:
Added a description for consistency with the native Grafana AdHoc filters (e.g. see [Explore Metrics](https://play.grafana.org/explore/metrics/trail?from=now-1h&to=now&var-ds=grafanacloud-demoinfra-prom&var-otel_resources=&var-filters=&refresh=&var-deployment_environment=&metricPrefix=all))